stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         release: {{ .Release.Name }}
       annotations:
         checksum/binarystore: {{ include (print $.Template.BasePath "/artifactory-binarystore.yaml") . | sha256sum }}
+        checksum/access-creds: {{ include (print $.Template.BasePath "/access-bootstrap-creds.yaml") . | sha256sum }}
```

Comment:
You need to add an if statement to check if no pre existing secret was provided. This file is not always used for the access bootstrap